packages/admin/admin/src/section/SectionHeadline.tsx

```diff
@@ -0,0 +1,146 @@
+import { Info } from "@comet/admin-icons";
+import { type ComponentsOverrides, Divider, Typography } from "@mui/material";
+import { css, type Theme, useThemeProps } from "@mui/material/styles";
+import { isValidElement, type ReactNode } from "react";
+
+import { Tooltip } from "../common/Tooltip";
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { type ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
+
+export type SectionHeadlineClassKey = "root" | "header" | "titleContainer" | "headline" | "divider" | "supportText" | "infoTooltip";
+
+const Root = createComponentSlot("div")<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "root",
+})();
+
+const Header = createComponentSlot("div")<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "header",
+})(
+    () => css`
+        display: flex;
+        flex-direction: row;
+        align-items: flex-end;
+        justify-content: space-between;
+    `,
+);
+
+const TitleContainer = createComponentSlot("div")<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "titleContainer",
+})(
+    () => css`
+        display: flex;
+        align-items: center;
+    `,
+);
+
+const Headline = createComponentSlot(Typography)<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "headline",
+})();
+
+const SupportText = createComponentSlot(Typography)<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "supportText",
+})(
+    ({ theme }) => css`
+        color: ${theme.palette.text.secondary};
+    `,
+);
+
+const InfoTooltip = createComponentSlot(Tooltip)<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "infoTooltip",
+})(
+    ({ theme }) => css`
+        color: ${theme.palette.text.secondary};
+        margin-left: 10px;
+    `,
+);
+
+const StyledDivider = createComponentSlot(Divider)<SectionHeadlineClassKey>({
+    componentName: "SectionHeadline",
+    slotName: "divider",
+})(
+    () => css`
+        margin-top: 10px;
+        color: "inherit";
```

Comment:
I don't think this works, as `"inherit"`(with quotes) is not valid in CSS... do you really need to set the color, if it looks correctly with this invalid value? 🤔